Documentation/kubernetes/install/eks.rst

```diff
@@ -54,18 +54,32 @@ Create EKS Cluster
 
        kubectl -n kube-system set env ds aws-node AWS_VPC_K8S_CNI_EXTERNALSNAT=true
 
-#. Assign a fixed security identity to kube-dns
+#. Assign a fixed security identity to ``kube-dns`` by  adding the label ``io.cilium.fixed-identity: kube-dns``
 
    .. code:: bash
 
-       kubectl label -n kube-system pod $(kubectl -n kube-system get pods -l k8s-app=kube-dns -o jsonpath='{range .items[]}{.metadata.name}{" "}{end}') io.cilium.fixed-identity=kube-dns
+       # if using kube-dns
+       kubectl patch -n kube-system deployment/kube-dns --type merge -p '{"spec":{"template":{"metadata":{"labels":{"io.cilium.fixed-identity":"kube-dns"}}}}}'
+       
+       # if using coredns
+       kubectl patch -n kube-system deployment/coredns --type merge -p '{"spec":{"template":{"metadata":{"labels":{"io.cilium.fixed-identity":"kube-dns"}}}}}'
 
-   This steps allows Cilium to bring up kube-dns networking and enforce
-   security policies before etcd is up.
+   This step allows Cilium to bring up kube-dns networking and enforce
+   security policies before etcd is up. (Note: By default, kubernetes keeps the old replicaset 
```

Comment:
kubernetes --> Kubernetes

replicaset --> ReplicaSet (maybe surround in \`\`)?